src/external-server/venv-devnet.ts

```diff
@@ -27,7 +30,8 @@ export class VenvDevnet extends ExternalServer {
 
     protected async spawnChildProcess(): Promise<ChildProcess> {
         const args = ["--host", this.host, "--port", this.port].concat(this.args || []);
-        return spawn(this.command, args);
+        const options = { env: { ...process.env, STARKNET_DEVNET_CAIRO_VM: this.vmLang } };
```

Comment:
Do we need to pass all the process env vars?

Reply:
https://nodejs.org/api/child_process.html#child_processspawncommand-args-options
looks like default value is process.env,
i made a change to only add PATH